pkg/engine/response/response.go

```diff
@@ -65,6 +65,8 @@ type RuleResponse struct {
 	Success bool `json:"success"`
 	// statistics
 	RuleStats `json:",inline"`
+	// PathNotPresent indicates whether referenced path in variable substitution exist
+	PathNotPresent bool `json:"pathNotPresent"`
```

Comment:
Curious why we introducing a check for a path not present, as this could rule that failed with an error message. 
So now for some errors, we have rule failure and message, and some new field added to a struct. (seems inconsistent design)

Reply:
From my understanding, 
1. a rule failure refers to a rule application failure, while the path not present is much more a policy definition error
2. since later we will have policy error reported as new CRD, it's more clear to have that 2 logic decoupled for the change.